src/main/java/com/esri/samples/ogc/open_street_map_layer/README.md

```diff
@@ -0,0 +1,18 @@
+# OpenStreetMap Layer
+Add the OpenStreetMap layer to a map as a Basemap.
+
+![OpenStreetMap Layer App](openstreetmap-layer.png)
+
+## How to use the sample
+Simply run the app.
+
+## How it works
+Instantiate an `ArcGISMap` with a `Basemap.Type.OPEN_STREET_MAP` and add it to a `MapView`. The attribution text that is required by OpenStreetMap is automatically added to the `MapView's` attributionText without any additional code.

```

Comment:
This is always shown as a list for Java SE. Make this a two step list with the first step to create the map and the second to set it to the map view. The part about the attribution can be omitted.